src/node/rpc/member_frontend.h

```diff
@@ -1824,18 +1825,30 @@ namespace ccf
       members(&network.members)
     {}
 
+    std::optional<tls::Pem> resolve_caller_id(
+      ObjectId caller_id, kv::Tx& tx) override
+    {
+      auto members_view = tx.get_view(*members);
+      auto caller = members_view->get(caller_id);
+      if (!caller.has_value())
+      {
+        return std::nullopt;
+      }
+
+      return caller.value().cert;
+    }
+
     bool lookup_forwarded_caller_cert(
```

Comment:
@jumaffre I tried to de-dupe those, but it turns out that their default behaviour is different from the base impl in frontend.h (return false), so that doesn't really work. I'll ticket the corresponding refactoring.

Reply:
#1877